### ISSUE TITLE
regression - pluck not working with translatable field

### ISSUE THREAD

**Issue by basex** [globalize/globalize#526]
There was a regression on the master probably on the last commits to make globalize work with ActiveRecord 5.

ActiveRecord 4.2.6
Globalize: 5.0.1

```ruby
PaymentMethod.includes(:translations).pluck(:name) 
#=> ['Paypal', 'Stripe', 'AliPay', 'FastSpring', 'BrainTree']
```

ActiveRecord 5.0
Globalize: master

```ruby
PaymentMethod.includes(:translations).pluck(:name) 
#=> ActiveRecord::StatementInvalid: Mysql2::Error: Unknown column 'payment_methods.name' in 'field list': SELECT `payment_methods`.`name` FROM `payment_methods` LEFT OUTER JOIN `payment_method_translations` ON `payment_method_translations`.`payment_method_id` = `payment_methods`.`id` WHERE `payment_methods`.`id` IN (250, 9435, 46210, 46662, 48421)
```

The query seems correct, the only problem is getting the correct column.

**Comment by davidwessman** [comment]
You can do it like:
`PaymentMethod::Translation.pluck(:name)` and it should work correctly

**Comment by basex** [comment]
There are several ways around, in my case I used map. The point is that it's a regression from the previous release.

**Comment by shioyama** [comment]
I assume you mean Globalize *5.0.1* right? There is no 5.1.0.

**Comment by basex** [comment]
Thanks. Corrected

**Comment by shioyama** [comment]
Ok, I *think* I've identified the source of the problem, [here](https://github.com/ballistiq/globalize/blob/d757e11f5f9f713937902331266b9ca0b92557cc/lib/globalize/active_record/act_macro.rb#L13). This has some Rails 5 specific code to add a virtual attribute which seems to break this use of `pluck`.

**Comment by shioyama** [comment]
Here's the commit: 699c4442. Revert it and `pluck` works as expected.

**Comment by shioyama** [comment]
Here's [the PR](https://github.com/globalize/globalize/pull/512) where it was added. We'll have to find another way to fix the original issue.

**Comment by shioyama** [comment]
Closed this, fixed in #555. Can you try the latest master and confirm that the issue is fixed? Thanks.

**Comment by basex** [comment]
Thanks @shioyama I confirm the problem is solved 👍
